test/testcafe/spec/IdentifyWithRememberUsername_spec.js

```diff
@@ -82,42 +82,6 @@ async function setup(t, options) {
   return identityPage;
 }
 
-test.requestHooks(identifyRequestLogger, identifyWithError)('identifer first flow - should NOT remember username after failed authentication', async t => {
```

Comment:
identifier is now stored in the cookie, regardless of authentication status (success/fail)